### PR TITLE
contractor mod now protects underclothes, ft. typo fixes

### PR DESCRIPTION
**Pull request by Hatterhat**
## About The Pull Request
image
(also includes some grammar fixes and added wound armor values for parity with other syndicate uniforms)

## How This Contributes To The Skyrat Roleplay Experience
i think if you're on fire in a sealed suit the fire should only be affecting your armor and not your pants

## Changelog

:cl:
code: Removed some redundant Syndicate underclothes' armor values.
fix: All Syndicate undersuit items now have wound armor matching their base tg variants.
fix: Cut down on some of the copy-pasted code regarding the Enclave uniforms, which are apparently in the donator tab.
spellcheck: Cleaned up some typos in Syndicate uniforms' descriptions.
/:cl:

### CODE REVIEW

File: modular_skyrat/master_files/code/modules/clothing/under/syndicate.dm [hunk 1]
```diff
@@ -124,37 +128,30 @@
...
 *	ENCLAVE
 */
-/obj/item/clothing/under/syndicate/skyrat/enclaveo
-	name = "neo american officer uniform"
-	desc = "Throughout the stars, rumors of mad scientists and angry drill sergeant run rampent, of creatures in armor black as night being led by men or women wearing this uniform, they share one thing, a deep, natonalistic zeal of the dream of America."
-	icon_state = "enclaveo"
-	can_adjust = TRUE
-	armor_type = /datum/armor/clothing_under/none
-
 /obj/item/clothing/under/syndicate/skyrat/enclave
```
Comment (ghost) [inline modular_skyrat/master_files/code/modules/clothing/under/syndicate.dm:116]: ~~this is unused and can likely be fully removed~~ nevermind for some cursed reason its a donor outfit. I still say removal though considering that it's just a fallout nazi faction's uniform

File: modular_skyrat/master_files/code/modules/clothing/under/syndicate.dm [hunk 2]
```diff
@@ -80,7 +83,7 @@
 // (Multiple non-syndicate things use the base tactical turtleneck, they cant have it red nor reskinnable. OUR version, however, can be.)
 /obj/item/clothing/under/syndicate/skyrat/tactical
 	name = "tactical turtleneck"
-	desc = "A snug syndicate-red turtleneck with carcoal-black cargo pants. Good luck arguing allegiance with this on."
```
Comment (ghost) [inline modular_skyrat/master_files/code/modules/clothing/under/syndicate.dm:83]: i made a typo 💀

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
I feel like that's not really the solution, fire just shouldn't spread to the clothes underneath MODsuits, that doesn't really make sense to me.

**Comment by lessthnthree** [comment]
Is this needed since https://github.com/tgstation/tgstation/pull/77694

**Comment by Hatterhat** [comment]
> Is this needed since [tgstation/tgstation#77694](https://github.com/tgstation/tgstation/pull/77694)

that only affects the syndicate modsuit but i guess i could extend it to contractor mods and see if that fixes it

it does not

File: modular_skyrat/master_files/code/modules/clothing/under/syndicate.dm [hunk 3]
```diff
@@ -5,13 +5,11 @@
 /obj/item/clothing/under/syndicate
 	worn_icon_digi = 'modular_skyrat/master_files/icons/mob/clothing/under/syndicate_digi.dmi' // Anything that was in the syndicate.dmi, should be in the syndicate_digi.dmi
 
-
 /obj/item/clothing/under/syndicate/skyrat
 	icon = 'modular_skyrat/master_files/icons/obj/clothing/under/syndicate.dmi'
 	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/under/syndicate.dmi'
 	//These are pre-set for ease and reference, as syndie under items SHOULDNT have sensors and should have similar stats; also its better to start with adjust = false
 	has_sensor = NO_SENSORS
-	armor_type = /datum/armor/clothing_under/syndicate_skyrat
```
Comment (GoldenAlpharex) [inline modular_skyrat/master_files/code/modules/clothing/under/syndicate.dm:14]: Why did this have to be removed, anyway?

Comment (Hatterhat) [reply modular_skyrat/master_files/code/modules/clothing/under/syndicate.dm:14]: it was the same values as the base tactical turtleneck and i thought it should just inherit from the parent. if you want i guess i could put it back but i don't really see the value in doing that

Comment (GoldenAlpharex) [reply modular_skyrat/master_files/code/modules/clothing/under/syndicate.dm:14]: Fair.

**Review by GoldenAlpharex — APPROVED** [review]

**Pull request merged**